src/file-events/cpp/generic_fsnotifier.cpp

```diff
@@ -175,20 +175,14 @@ void AbstractServer::reportChange(JNIEnv* env, int type, const u16string& path)
         env->ExceptionClear();
 
         jclass exceptionClass = env->GetObjectClass(exception);
-        assert(exceptionClass != nullptr);
```

Comment:
Can we remove the `#include <assert.h>` statements as well?

Reply:
Getting rid of them as well now.